.travis.yml

```diff
@@ -1,4 +1,17 @@
 language: generic
-before_install: rvm reinstall ruby-head --binary
+addons:
+  apt:
+    packages:
+      - python3
+before_install:
+  - rvm reinstall ruby-head --binary
+  - jdk_switcher use oraclejdk8
+  - ./.travis/install_from_source
 install: gem install jekyll
-script: jekyll build
+before_script: rm ./googlecfd75a4bac031411.html
+script: 
+  - jekyll serve --detach --no-watch
```

Comment:
The script part shouldn't be outsourced to another script, because travis treats each line as individual check.
Currently you can easily spot like: oh, check 4 failed, all the others seem ok. I must have mistyped a link somewhere...
And therefore we only need a single script. I agree, the name is probably not the best ^^